cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaReconciler.java

```diff
@@ -808,6 +861,119 @@ protected Future<Void> updateCertificateSecrets(List<Secret> secrets) {
         return Future.join(reconcileFutures).mapEmpty();
     }
 
+
+    /**
+     * Copies secrets for trusted certificates used by AuthZ into a single internal secret.
+     * This is used when configuring truststore for the AuthZ server.
+     *
+     * @return  Future that completes when the internal Secret was successfully created or updated
+     *          or when Authz is not used or no trusted certificates specified
+     */
+    @Deprecated
+    protected Future<Void> authzTrustedCertsSecret() {
+        Set<String> secretsToCopy = new HashSet<>();
+        List<Integer> certHashes = new ArrayList<>();
+
+        // opa is deprecated?
```

Comment:
yes it was deprecated in 0.46.0 but I guess we still need this part until it's removed.